tools/run_perf_builder.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Attic Labs, Inc. All rights reserved.
+# Licensed under the Apache License, version 2.0:
+# http://www.apache.org/licenses/LICENSE-2.0
+
+'''This script runs the Noms PR Jenkins jobs on http://jenkins-perf.noms.io:
+- http://jenkins-perf.noms.io/job/NomsMasterPerf
+- http://jenkins-perf.noms.io/job/NomsPRPerf
+'''
+
+import copy
+import os
+import os.path
+import re
+import subprocess
+
+# These are the 'go test' packages for perf testing.
+# Note that adding entires will actually run all tests in that package, not just the perf tests.
+PACKAGES = [
+    './samples/go/csv/csv-import',
+]
+
+# 'go test' timeout. Go's default is 10m, which ideally is long enough, but raise it a little.
+TIMEOUT = '15m'
+
+# Number of perf test repetitions. Choose 4 because:
+# - Any less than 3 is not enough sample size.
+# - Any more than 4 will take too long.
+# - Typically the first test will take longer, so choosing 4 gives you 3 non-first samples.
```

Comment:
The compilation of the binaries actually shouldn't count towards the "elapsed" metric, only the total.

But yes, other things. From the point of view of somebody that doesn't know test specifics, the first test might be slower because the OS hasn't cached files (e.g. the CSV off disk). Or the startup time for those binaries would be slower.

However there is actually a test that loads a CSV off disk, into a blob, then writes that blob to the database. Subsequent runs are much faster because the blob is already in the database, a short circuit must be tripping somewhere.